src/libs/theme.js

```diff
@@ -187,11 +187,11 @@ export const Styles = {
       fontFamily: 'Montserrat',
     },
     CARDCONTAINER: {
-      marginTop: '2rem',
+      margin: '2rem -12.5%',
```

Comment:
Can we tighten up the border between the top of the opened table with the row above it? Locally, I was able to get those to line up with `margin: '-.75rem -12.5%',`. However, `rem` is somewhat flexible and could change in different browsers. Might be nice to modify the border radius on the top of the table and bottom of the summary row to make them look even more tightly coupled.

## Original
![Screenshot 2023-02-09 at 7 17 04 AM](https://user-images.githubusercontent.com/116679/217811638-51450c29-a14d-4d82-8061-12172cb803c0.png)

## Updated
![Screenshot 2023-02-09 at 7 22 17 AM](https://user-images.githubusercontent.com/116679/217811832-d8b1ea28-236e-4aec-baa6-04ae0bebaed8.png)